### PR TITLE
[Improvement-15183][api] remove ApiException annotation

### PR DESCRIPTION
**Pull request by zhengkezhou1**
## Purpose of the pull request

the **@ApiException** is no need
this pull request slove https://github.com/apache/dolphinscheduler/issues/15183

## Brief change log

- change **org/apache/dolphinscheduler/api/exceptions** 
- change **org/apache/dolphinscheduler/api/controller**  
- change **org/apache/dolphinscheduler/api/controller/v2**
- change **test/org/apache/dolphinscheduler/api/exceptions**
## Verify this pull request

This pull request is already covered by existing tests, such as **unit test**

### CODE REVIEW

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java [hunk 1]
```diff
@@ -325,41 +323,23 @@ public static String getFileChecksum(String pathName) throws IOException {
...
-            if (0 != exitCode) {
-                throw new FileOperateException(
-                        "Set file: " + filePath + " to owner: " + fileOwner + " failed, existCode(" + exitCode + ")");
-            }
-        } catch (FileOperateException ex) {
-            throw ex;
-        } catch (Exception ex) {
-            throw new FileOperateException("Set directory: " + filePath + " to owner: " + fileOwner + " failed");
-
+    public static void setFileOwner(Path filePath, String fileOwner) throws InterruptedException, IOException {
```
Comment (ruanwenjun) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java:326]: Why you change this file?

**Review by ruanwenjun — COMMENTED** [review]
Basically LGTM, please rebase from the upstream-dev, it seems you have change some files which is shouldn't happen in this PR.

**Comment by zhengkezhou1** [comment]
ok, pls see PR: https://github.com/apache/dolphinscheduler/pull/15291

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java (hunk 1 above)
Comment (zhengkezhou1) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java:326]: 图片
sorry, i made a mess,  i need to learn how to use git

**Comment by ruanwenjun** [comment]
closed by #15291

**Pull request closed without merging**